packages/trpc/server/routers/viewer/teams/inviteMember/utils.ts

```diff
@@ -154,20 +154,45 @@ export async function createNewUserConnectToOrgIfExists({
       ? slugify(emailUser)
       : slugify(`${emailUser}-${emailDomain.split(".")[0]}`);
 
+  // Check if there is another user with the same email to act as a parent
+  const parentUser = await prisma.user.findFirst({
+    where: { email: usernameOrEmail, linkedByUserId: null },
+    select: { id: true, password: true, emailVerified: true },
+  });
+
   const createdUser = await prisma.user.create({
     data: {
       username,
       email: usernameOrEmail,
       verified: true,
       invitedTo: input.teamId,
-      organizationId: orgId || null, // If the user is invited to a child team, they are automatically added to the parent org
+      emailVerified: parentUser?.emailVerified,
+      password: parentUser?.password,
+      ...(orgId
+        ? {
+            organization: {
+              connect: {
+                id: orgId,
+              },
+            },
+          }
+        : {}),
```

Comment:
Reusing parent info as the email may be already be verified, and the password should be the same one as the parent user.